model/articles.model.js

```diff
@@ -23,3 +23,13 @@ exports.patchVotes= (article_id,votes)=>{
     })
 }
 
+exports.fetchAllArticles= () =>{ 
+    return db.query("SELECT articles.author,articles.title,articles.article_id,articles.topic,articles.created_at,articles.votes,COUNT(comments.comment_id) ::INT AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY created_at DESC").then((results)=>{
```

Comment:
Please, for the sake of my eyes, break this out onto multiple lines (it'll let you if you use ` instead of ")

Reply:
yeah will do! 😂😂